lib/spryte/rspec/macros.rb

```diff
@@ -12,7 +12,8 @@ def request(name)
 
         host ENV.fetch("SPRYTE_RSPEC_HOST", "localhost")
 
-        let(:method) { :get }
+        let(:through) { :get }
+        let(:method) { method(:get) }
```

Comment:
Actually, thinking about it, shouldn't this code also be just calling 

```ruby
through(:get)
```

In place of 
```ruby
let(:through) { :get }
```

On the basis of DRYness: currently both the `request` and `through` methods know how to set those `let`s, which is duplicated knowledge.